File: Assignment_39/Assignment_39_3.java
class Demo
{
    public void display(int iNo)
    {
        int iCnt = 0;
        //O(n) 
        for(iCnt = iNo; iCnt >= 1 ; iCnt--)
        {
            System.out.print(iCnt+"\t");
        }
        System.out.println("");
    }
}

class Assignment_39_3 
{
    public static void main(String A[])
    {
        int iValue = 5;

        Demo dobj = new Demo();

        dobj.display(iValue);
    }
    
}

File: Assignment_39/Assignment_39_3.c
#include<stdio.h>

void Display(int iNo)
{
    int iCnt = 0;
    //O(n) 
    for(iCnt = iNo; iCnt >= 1 ; iCnt--)
    {
        printf("%d\t",iCnt);
    }
    printf("\n");
}

// Main Function
int main()
{

    int iValue = 5;
    
    Display(iValue);

    return 0;
}